src/esp/assets/FRLInstanceMeshData.cpp

```diff
@@ -18,8 +18,11 @@
 #include <vector>
 
 #include <fcntl.h>
+#ifndef _WIN32
 #include <sys/mman.h>
 #include <unistd.h>
+#endif
```

Comment:
These seem to be just copy-paste leftovers from `PTexMeshData.cpp`, so could be removed completely I think.

Reply:
removed.